radicle-cli/src/commands/patch.rs

```diff
@@ -79,6 +94,14 @@ pub struct Options {
     pub verbose: bool,
 }
 
+fn parse_patch_id(val: OsString) -> Result<OptPatch, anyhow::Error> {
+    let val = val
+        .to_str()
+        .ok_or_else(|| anyhow!("patch id specified is not UTF-8"))?;
+    let patch_id = PatchId::from_str(val).map_err(|_| anyhow!("invalid patch id '{}'", val))?;
+    Ok(OptPatch::Patch(patch_id))
+}
```

Comment:
It seems like we could move this function and the issue-id parsing one into a `terminal::parse` module or something, and combine the functionality into a single function.

Reply:
Yes and likewise other ID parsing commands so the messages are consistent.

Would we be happy in putting it in `terminal` until the context is clearer?  Its still only 174 lines long.  It may be things in `terminal` are better organized by domain then function, for instance we have `terminal::patch` already.

Reply:
I think it would maybe be `terminal::cob` then, since it applies to all cobs.